src/csharp/Grpc.Examples/MathServiceImpl.cs

```diff
@@ -1,6 +1,6 @@
 #region Copyright notice and license
 
-// Copyright 2015, Google Inc.
+// Copyright 2015-2016, Google Inc.
```

Comment:
That's required by our copyright checker - it sees the file has been touched in git history in 2016 and demands updating the copyright.